tests/docker/test_parsing.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from prefect.docker import parse_image_tag
+from itertools import product
+
+image_names = ['some_url_to_a_docker_repo.com:5050/path/to/image','some_url_to_a_docker_repo.com/path/to/image']
+image_tags = [[], ['tag1']]
+@pytest.mark.parametrize('docker_image,image_name,image_tag', [(':'.join([image]+tag), image, tag) for image, tag in product(image_names, image_tags)])
+def test_docker_image_parsing(docker_image,image_name,image_tag):
+    parsed_image, parsed_tag = parse_image_tag(docker_image)
+    assert parsed_image == image
```

Comment:
This line raises an error `NameError: name 'image' is not defined`. 

Just in case you're not familiar with our testing setup, you can install and test the forked version of Prefect by starting a virtual environment, going into the root of the repo, and running `pip install -e ".[dev]"`, and then running `pytest tests/docker/test_parsing.py `. It's also worth running pytest for any other code that yours might impact, so `pytest tests/docker` and `pytest tests/infrastructure/test_docker_container.py`.

Reply:
One of your cases fails when parsed (the second case), so that's probably worth checking out in the function itself.
```
-> assert parsed_image == image_name
(Pdb) parsed_image
'5:0:5:0:/:p:a:t:h:/:t:o:/:i:m:a:g:e'
```